RELEASE.md

```diff
@@ -0,0 +1,195 @@
+# sqlparse - LALR grammar based SQL Parser
```

Comment:
Please consider moving file to wiki. Since we don't package sqlparse release notes a link from README pointing to release notes wiki should be enough!